src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -6382,27 +6382,28 @@ bool Unit::HandleDummyAuraProc(Unit* victim, uint32 damage, AuraEffect* triggere
                 }
                 else
                 {    // Check Party/Raid Group
-                    if (Group* group = ToPlayer()->GetGroup())
```

Comment:
@Goatform The line will never be reached for pets, there is a check for typeid being `TYPEID_PLAYER` further up.